src/backer/cardaning.py

```diff
@@ -129,9 +123,10 @@ def rollbackTrans(self, tipHeight, rollBackSlot):
                 blockSlot = item["block_slot"]
 
                 if blockSlot > rollBackSlot:
```

Comment:
Maybe you can check ogmios documentation here. Does it rollback block by block or rollback TO a slot (like just X slots back)

I'm not sure if this should be > or == on this line.

For example, I think it COULD be the case that:
- on slot 500
- rollback to slot 470
- the block in the rollback event is the new slot 470

Please check the docs and see. If that's true, then we'd need to resubmit with == I think because the slot 470 block has been replaced and is different to the original one in the forward direction.

Reply:
ah, you are right, the response from Ogmios only return 1 Block for backward direction ([here](https://gitlab.com/viperscience/ogmios-python/-/blob/main/src/ogmios/chainsync/NextBlock.py?ref_type=heads#L65)), I will change to "=="

Reply:
Yes but have you verified if going from 500 to 499 in the backwards direction means:
1. We remove block 500, and have the same 499
2. We remove block 500, 499 and inserted a NEW 499

?

Reply:
still have not gotten exactly the point in that, but after reading some documentation I think the "1" is correct So I think we need to resumit all of our old block after 500, I mean we use operation '>' instead of "=", ">="

Reply:
@Sotatek-Patrick-Vu Looking at the API, when we rollback we don't get a block. We get a point. See https://ogmios.dev/api/#operation-publish-/?FindIntersection

I think on a rollback, `block` will be `None`.

Which aligns with the doc you sent me https://ogmios.dev/mini-protocols/local-chain-sync/#requesting-next-blocks

So instead of `block`, we have a `point` object.

Reply:
This point object is the last known point which is VALID. So if the point object was pointing at block 499, I believe we still keep 499. So this is scenario number 1.

So yes, we resubmit everything AFTER 499

Reply:
yep, exactly what I want to process for our situation here :)

Reply:
yep, just make sure we are capturing the `point` object too instead of None =)

Reply:
fixed, I still keep the name "block" it is a point